tests/flux-automation.sh

```diff
@@ -13,12 +13,13 @@ whitelist_dirs=(
     monitoring
     neuvector)
 
-# Applications that are currently excluded from this check 
+# Applications that are currently excluded from this check
 exclusions=(
   k8s/aat/common/em/em-showcase.yaml
   k8s/aat/common/idam/idam-api.yaml
   k8s/aat/common/idam/idam-web-admin.yaml
   k8s/aat/common/idam/idam-web-public.yaml
+  k8s/aat/common/sscs/sscs-tya-frontend.yaml
```

Comment:
why are you adding it here?

Reply:
Failing automated tests. I'm removing the non prod instances of TYA-Frontend service, then will be removing the prod definition and this exclusion once proven on AAT.

Reply:
k